src/medals/events.cljs

```diff
@@ -16,11 +16,30 @@
    (println selected-sort)
    (assoc db :sort selected-sort)))
 
+
+
+(re-frame/reg-event-db
+ ::fetch-countries-success
```

Comment:
nice job using `::` instead of `:`. I'm not sure if you purposefully did this or are mimicking examples you've seen but I want to call it out as best practice since it includes the namespace of the file it's in. Here's [a post](https://blog.jeaye.com/2017/10/31/clojure-keywords/) with it more clearly explained.

Reply:
Yes That's was the sole reason I have used because it gives us the namespaced keyword of the current namespace.